packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/useCursorUtils.ts

```diff
@@ -89,6 +89,7 @@ export default function useCursorUtils(CodeMirror: any) {
 
 				for (let j = 0; j < lines.length; j++) {
 					const line = lines[j];
+					if (!line) continue;
```

Comment:
Maybe it should be `break` instead?
What'd happen if you select the text below and convert it to a list?
```
list
list


not list

```

Reply:
> What'd happen if you select the text below and convert it to a list?
## After the fix:
### Case 1: Only 1st and 2nd lines are selected
 Both 1st and 2nd lines will be converted to list; 

### Case 2: All four lines are selected
"All except third (empty)"  line will be converted to list.

> Before fix : in case 2 above; all 4 lines will be converted to list

Reply:
> Maybe it should be `break` instead?
I'm not sure; It depends on what is the expected behaviour for the case you described?

Reply:
I just checked the behaviour of google docs and microsoft word. It looks like Word behaves with the continue behaviour. Google docs just adds a bullet to every line. 

I have no personal preference, but someone else might. It could probably be worth putting this to a poll on the forum.

Reply:
[LInk](https://discourse.joplinapp.org/t/expected-behaviour-when-creating-list-using-toolbar/16683) to the poll on joplin forum.